pkg/operation/seed/seed.go

```diff
@@ -385,6 +385,10 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 		lokiValues["authEnabled"] = false
 		lokiValues["hvpa"] = map[string]interface{}{
 			"enabled": hvpaEnabled,
+			"maintenanceTimeWindow": map[string]interface{}{
+				"begin": "220000-0000",
+				"end":   "230000-0000",
```

Comment:
Does it make sense to synchronize this with the shooted seed's maintenance window? Is it even possible?

Reply:
It makes sense if it's a shooted seed, yes. You can try reading the `kube-system/shoot-info` config map. If it exists then you'll find the maintenance time window in its data:

```yaml
apiVersion: v1
data:
  maintenanceBegin: 120000+0000
  maintenanceEnd: 130000+0000
  ...
kind: ConfigMap
metadata:
  annotations:
    resources.gardener.cloud/description: |-
      DO NOT EDIT - This resource is managed by gardener-resource-manager.
      Any modifications are discarded and the resource is returned to the original state.
  creationTimestamp: "2019-12-10T12:10:25Z"
  labels:
    shoot.gardener.cloud/no-cleanup: "true"
  name: shoot-info
  namespace: kube-system
  resourceVersion: "307696014"
  selfLink: /api/v1/namespaces/kube-system/configmaps/shoot-info
  uid: 0c3404b0-1b46-11ea-894d-5e2e6521d6ea
```

, so you can take these values. If it does not exist then what about computing a random time window here?